personnel.md

```diff
@@ -0,0 +1,216 @@
+# Personnel
+
+## Current
+
+### Secretaries
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| Alessandro Lai ([@AlessandroLai])     | 2017-11-12 - 2018-08-31 |
+| Margret Staples ([@dead_lugosi])      | 2017-11-12 - 2019-05-31 |
+| Michael Cullum ([@michaelcullumuk])   | 2016-01-31 - 2018-01-31 |
+
+Feel free to contact the secretaries at info AT php-fig.org. For more information on the secretaries' role, see the [bylaws](/bylaws).
```

Comment:
You should link to the same link as now, `/bylaws/mission-and-structure/#secretaries`